openedx/core/djangoapps/ace_common/templates/ace_common/edx_ace/common/base_body.html

```diff
@@ -33,6 +33,7 @@
     margin: 0;
     padding: 0;
     min-width: 100%;
+    background-color: #f5f5f5;
```

Comment:
we wanted a background color for this email to match our other marketing emails but Kavya said we could apply this to all our emails